client/gutenberg/extensions/map-block/add-point/style.scss

```diff
@@ -0,0 +1,53 @@
+/** @format */
+
+.component__add-point {
+	position: absolute;
+	left: 50%;
+	top: 50%;
+	z-index: 1;
+	width: 32px;
+	height: 38px;
+	margin-top: -19px;
+	margin-left: -16px;
+	background-image: url( https://s3.amazonaws.com/atavist-static/map/oval.svg );
```

Comment:
Just a note that we will need to move this file to be loaded from the Jetpack side directly.

Reply:
There was discussion of moving all the static assets to WP.com ( https://github.com/Automattic/wp-calypso/pull/27836#discussion_r226570934 ).

Reply:
Resolved in https://github.com/Automattic/wp-calypso/pull/27836/commits/ea07f2e853409425716133aed494a53afd9d2107